L13/index.html

```diff
@@ -11,12 +11,19 @@ <h2>L13 task</h2>
 	<p>
 		Create html page. And implement a traffic light on it. The traffic light must have 3 lights (green, yellow and red). Light bulbs should constantly switch without any interaction from the user of the page. <br/><br/><br/> P.S. The rules for switching light bulbs should at least remotely resemble a real traffic light.
 	</p>
-	<div class="semafore">
-		<div id="first" class="circle redlight"></div>
-		<div id="second" class="circle yellowlight"></div>
-		<div id="third" class="circle greenlight"></div>
+	<div class="semafors">
```

Comment:
just a recommendation to using couple words for example change the `semaphores` to `semaphore-list`